resources/definitions/tizyx_evy.def.json

```diff
@@ -57,7 +57,7 @@
                 [25, 49],
                 [25, -49],
                 [-25, -49],
-                [25, 49]
```

Comment:
aren't these change unrelated the ticket? Can we perhaps move these changes to another ticket?

Reply:
@saumyaj3 That was my original intention, but @casperlamboo told me to include it as it is a quick fix and doesn't require a dedicated ticket.

Reply:
Alright then! This looks good :)